docs/UserGuide.adoc

```diff
@@ -28,32 +32,33 @@ Having any Java 8 version is not enough. +
 This app will not work with earlier versions of Java 8.
 +
 .  Download the latest `addressbook.jar` link:{repoURL}/releases[here].
-.  Copy the file to the folder you want to use as the home folder for your Address Book.
+.  Copy the file to the folder you want to use as the home folder for CLIndar.
 .  Double-click the file to start the app. The GUI should appear in a few seconds.
 +
-image::Ui.png[width="790"]
+Please insert an image here
```

Comment:
Hmm why is it Please insert an image here? Should it be like *`Coming Soon in v2.0`* or something instead? or are we supposed to put an image here when we are done editing?

Reply:
Yup, we are supposed to put an image here